src/api/roles/coach/coach.py

```
@@ -233,24 +237,45 @@ def get_coach_availability(coach_id: int, db = Depends(get_session), acc: Accoun
+@router.get("/client_requests", response_model=RequestListResponse)
+def get_client_requests(db = Depends(get_session), acc: Account = Depends(get_coach_account)):
     """
-    Accepts a client coach request, creates a client coach relationship row
-    Errors when client coach request is not found, or if the request is not for a coach_id that matches the current user's coach_id
+    Gets the list of all client requests for a given coach
     """
+    if acc.coach_id is None:
+        raise HTTPException(404, detail="No coach profile found for this account")
     
-    request = db.query(ClientCoachRequest).filter(ClientCoachRequest.id == request_input.id).first()
+    requests = db.query(ClientCoachRequest).filter(ClientCoachRequest.coach_id == acc.coach_id, ClientCoachRequest.is_accepted == False).all()
```

Ruff (0.15.10)

[error] Avoid equality comparisons to `False`; use `not ClientCoachRequest.is_accepted:` for false checks

Replace with `not ClientCoachRequest.is_accepted`

(E712)